### ISSUE TITLE
Should demonstrate bounds checking with comparison operators going the same way

### ISSUE THREAD

**Issue by DontPanic345** [FreeCodeCamp/FreeCodeCamp#8439]
Challenge [Comparisons with the Logical And Operator](https://www.freecodecamp.com/challenges/comparisons-with-the-logical-and-operator#?solution=%0Afunction%20testLogicalAnd(val)%20%7B%0A%20%20%2F%2F%20Only%20change%20code%20below%20this%20line%0A%0A%20%20if%20(25%20%3C%3D%20val%20%26%26%20val%20%3C%3D%2050)%20%7B%0A%20%20%20%20%20%20return%20%22Yes%22%3B%0A%20%20%20%20%7D%0A%0A%20%20%2F%2F%20Only%20change%20code%20above%20this%20line%0A%20%20return%20%22No%22%3B%0A%7D%0A%0A%2F%2F%20Change%20this%20value%20to%20test%0AtestLogicalAnd(10)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.94 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```javascript

if (num > 5 && num < 10) {
  return "Yes";
}
return "No";

```
Is in the description text but it is much better to write comparisons how I have below with the comparison operators going the same way and 'val' in the middle

My code:
```javascript

function testLogicalAnd(val) {
  // Only change code below this line

  if (25 <= val && val <= 50) {
      return "Yes";
    }

  // Only change code above this line
  return "No";
}

// Change this value to test
testLogicalAnd(10);

```

**Comment by bcherny** [comment]
It's a matter of style, but unless you're writing C++, it's usually idiomatic put the *thing with the variable value* first in a comparison. That's because it sounds like how you would explain it in plain English.

Compare the following:

1. `5 < num && num < 10`: "5 is less than num and num is less than 10"
2. `num > 5 && num < 10`: "num is more than 5 and num is less than 10"

To me, the 2nd sounds more readable.

Also note that in some languages (eg. CoffeeScript), you can do stuff like `5 < num < 10` :)

**Comment by raisedadead** [comment]
IMHO, what you guys are suggesting is sort of style guide or best practice. I think this can be taken up as a wiki article, but not in the scope of the challenge, we do not want to confuse campers with things that while reasonable are overwhelming especially when they are new to coding.

/cc @FreeCodeCamp/issue-moderators

**Comment by ltegman** [comment]
I 100% agree that this is a matter of personal preference and you'll find just as many people who prefer it each way. For that reason we're going to leave it as-is since neither is 'correct'.

Thanks and happy coding!
